test/api_test/test_helper.rb

```diff
@@ -9,5 +9,7 @@
 
 require 'smart_proxy_dynflow'
 
+Proxy::Dynflow::Plugin.load_test_settings({})
```

Comment:
Our test suite was not gree due to this line missing, so the default config was not loaded with recent proxy